data/root/etc/inst_setup

```diff
@@ -192,10 +192,10 @@ zram_swap_on() {
 }
 
 zram_swap_off() {
-  if [ -b $zram_swap_dev ] ; then
-    swapoff $zram_swap_dev
-    echo $zram_dev_index > /sys/class/zram-control/hot_remove
-  fi
+  [ -b "$zram_swap_dev" ] || return
+
+  swapoff $zram_swap_dev
+  echo $zram_dev_index > /sys/class/zram-control/hot_remove
 }
```

Comment:
NP: not sure why you hate `if`

Reply:
> NP: not sure why you hate `if`

Me neither :stuck_out_tongue_winking_eye: 

Jokes aside, I prefer guard clauses or early return instead of nested code. That's true. Usually they make the code more readable/understandable (which might be not the case here). Anyway, I was simply trying to avoid a double nesting here because the code was something like 

```bash
if [ -b "$zram_swap_dev" ]; then
  swapoff $zram_swap_dev
  [ -e /sys/class/zram-control/hot_remove] && echo $zram_dev_index > /sys/class/zram-control/hot_remove
fi
```

but then I forgot to _restore_ the `if` clause after getting ride of the not longer needed file existence check.

I'll _fix_ it :)